wazo_call_logd/tests/test_cel_interpretor.py

```diff
@@ -277,11 +277,14 @@ def _assert_that_interpret_cel_calls(self, function, eventtype):
 class TestCallerCELInterpretor(TestCase):
     def setUp(self):
         self.caller_cel_interpretor = CallerCELInterpretor()
-        self.call = Mock(
+        self.call = create_autospec(
             RawCallLog,
+            instance=True,
             interpret_caller_xivo_user_fwd=True,
             extension_filter=Mock(filter=lambda x: x),
+            participants=[],
         )
+        assert hasattr(self.call, "participants")
```

Comment:
No need for an assert in setup code IMHO

Reply:
I added the assert because the `participants` field wasn't available to the mock(despite being part of the fields in the class interface, which I would expect the mock to know). 
It's not necessary anymore since I set `participants` explicitly in the mock.